app/views/devise/passwords/edit.html.haml

```diff
@@ -20,6 +20,6 @@
     = f.input_field :password_confirmation, class: 'form-control'
 
   .form-group
-    = f.button :submit, t('.change_password'), class: 'button'
+    = govuk_button() { t('.change_password') }
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3605/files#r526000031)